maps/kondaru.dmm

```diff
@@ -44119,8 +44111,7 @@
 /area/station/crew_quarters/cafeteria)
 "gVX" = (
 /obj/machinery/light/small{
-	dir = 8;
-	pixel_x = -12
+	dir = 8
```

Comment:
Not a fan of this light relocation. Could move the status display down two tiles instead.